zombies.js

```diff
@@ -137,8 +187,17 @@
  * @param {Item/Weapon/Food} item   The item to discard.
  * @return {boolean} true/false     Whether player was able to remove item from pack.
  */
-
-
+  discardItem(item){
```

Comment:
There is not validation to check if item is of type Item or it's children prototype.